arenadata/scripts/run_resgroup_test.bash

```diff
@@ -68,3 +94,4 @@ EOF1
             exit \$errcode
         )
 EOF
+save_logs
```

Comment:
do we call save_logs twice in case of non-zero return code form docker-compose?

Reply:
Yes. in line 97 if the tests were successful and in line 62 if they failed.

Reply:
I believe that line 97 will be executed in both cases and override logs from line 62. Won't it?

Reply:
the script is executed with the `-e` parameter, which says that we exit if any command ends with a non-zero code. Therefore, if docker-compose exec fails (tests failed), save_logs will executed only on line 62.